crawler/plugins/package_container_crawler.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+try:
+    from crawler.package_crawler import crawl_packages
+    from crawler.runtime_environment import IRuntimeEnvironment
+    from crawler.dockerutils import (exec_dockerinspect,
+                                     get_docker_container_rootfs_path)
+    from crawler.namespace import run_as_another_namespace, ALL_NAMESPACES
+    from crawler.misc import join_abs_paths
+    from crawler.crawler_exceptions import (CrawlError,
+                                            CrawlUnsupportedPackageManager)
+    from crawler.icrawl_plugin import IContainerCrawler
+except ImportError:
+    from package_crawler import crawl_packages
```

Comment:
in line with other common code being moved to plugins/ , shouldn't package_crawler.py also be moved from crawler/ to  crawler/plugins/?